Src/Base/AMReX_VisMF.cpp

```diff
@@ -1152,6 +1165,10 @@ VisMF::Write (const FabArray<FArrayBox>&    mf,
 
     delete whichRD;
 
+    if (Gpu::inLaunchRegion()) {
+        amrex::prefetchToDevice(mf);
```

Comment:
is `prefetchToDevice` a no-op if the mf uses host memory, or do we need to check for that here?

Reply:
Yes, see #842, which fixed this issue.  Prefetch is only supposed to be called on managed memory.

Reply:
I will modify this PR slightly.  For non-managed, we don't even need to start MFIter.